components/fraudlabs_pro/actions/send-sms-verification/send-sms-verification.mjs

```diff
@@ -0,0 +1,73 @@
+import  * as flpnodejs from 'fraudlabspro-nodejs';
+
+export default {
+  name: "Send SMS Verification",
+  description: "Send an SMS with verification code and a custom message for authentication purpose. Please refer to the [documentation](https://www.fraudlabspro.com/developer/api/send-verification) for the explanation of the result returned. NOTE: You need to register for an API key before using this REST API. Please visit [Micro Plan](https://www.fraudlabspro.com/sign-up) to sign up for an API key if you do not have one. In addition, you will also have to make sure you have enough SMS credits to send any verification SMS.",
+  key: "fraudlabs_pro-send-sms-verification",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    flp_api_key: {
+      type: "app",
+      app: "fraudlabs_pro",
+      description: "API license key. You can sign up for a trial key at [here](https://www.fraudlabspro.com/subscribe?id=1).",
+    },
+    tel: {
+      type: "string",
+      label: "tel",
+      description: "The recipient mobile phone number in E164 format which is a plus followed by just numbers with no spaces or parentheses. For example, `+123456789`",
+    },
+    mesg: {
+      type: "string",
+      label: "mesg",
+      description: "The message template for the SMS. Add <otp> as placeholder for the actual OTP to be generated. Max length is 140 characters. For example, `Your OTP for the transaction is <otp>`",
+    },
+    country_code: {
```

Comment:
please rename these properties in a lowerCase style in that way we are consistent with rest of  components.
```suggestion
    countryCode: {
```